docs/pages/reference/access-controls/roles.mdx

```diff
@@ -142,6 +144,52 @@ spec:
       'workload': ['database', 'backup']
 ```
 
+### `client_idle_timeout`
+
+This example configures a global client idle timeout of **30 minutes**. After 30 minutes of inactivity, the session will be terminated.
```

Comment:
It looks like this paragraph was left in by mistake.

Reply:
Oh I meant the example shown above on the page. I will clarify.

Reply:
Since we include examples later in this section, I think it makes sense to remove this paragraph to make this section a little shorter as well as to start the section with a general statement about the client idle timeout.